src/components/audio-zone-source.js

```diff
@@ -0,0 +1,55 @@
+import { THREE } from "aframe";
+
+const zero = new THREE.Vector3(0, 0, 0);
+
+/**
+ * Represents an audio-zone-entity that has an audio component in the audio-zones-system.
+ * Expects an audio-params component. It can be an audio, video, avatar or an audio target.
+ */
+AFRAME.registerComponent("audio-zone-source", {
```

Comment:
The he main reason to keep them separated is to be able to composite the entities using the components. In the case of the `audio-params` and `audio-zone-source` is not that clear as the only entity that has `audio-params` but not `audio-zone-source` is the avatar rig.

In the case of the `audio-zone-entity` it's also used by the `audio-zone-listener` as both source and listener need to keep a zones state.

But I'm happy to discuss it in case you see a clearer path to simplify this.